core/src/epicli/data/common/ansible/playbooks/roles/postgresql/defaults/upgrade.yml

```diff
@@ -0,0 +1,39 @@
+---
+pg_10:
```

Comment:
WDYT about naming in the way when it's not required updating this var name for the next upgrades?

Reply:
Refactored to use named include 👍🏻